src/libcore/slice.rs

```diff
@@ -655,6 +655,12 @@ impl<'a, T> Default for &'a [T] {
     fn default() -> &'a [T] { &[] }
 }
 
+#[stable(feature = "rust1", since = "1.5.0")]
+impl<'a, T> Default for &'a mut [T] {
+    #[stable(feature = "rust1", since = "1.5.0")]
```

Comment:
@Gankro As @alexcrichton said, although we don't need it - we have `mut_slice_default` instead.